### DISCUSSION TITLE
Activation email not working

### DISCUSSION THREAD

**Discussion by oasga** [Budibase/budibase#3573] [Help]
When I click the activation email it takes me to a page that says...
"Check your email. To get started with Budibase, click the verification link in the email we sent you."

It's caught in the loop. I verify and it sends me to the verification email sent page. 

How can I sign up to test out the app?

Thanks

**Comment by mjashanks** [comment]
Hey @oasga - I've just tried this, and the flow worked ok for me.

Could you send the email address that you signed up with to support@budibase.com - we can look into this. Thanks.
